src/test/java/org/apache/sling/commons/mime/internal/MimeTypeServiceImplTest.java

```diff
@@ -152,6 +165,29 @@ public void testProvider2() throws Exception {
         assertNull(this.service.getMimeType(GIF));
     }
 
+    public void testMultipleMimeTypes() throws Exception {
+        loadMimeTypes(MimeTypeServiceImpl.CORE_MIME_TYPES);
+        loadMimeTypes(MimeTypeServiceImpl.MIME_TYPES);
+
+        for (String mm : epsMimeTypeExt.keySet()) {
+            assertEquals("Extension " + mm + " (1)", epsMimeTypeExt.get(mm), this.service.getExtension(mm));
+        }
+    }
+
+    public void testNegativeTests() throws Exception {
+        loadMimeTypes(MimeTypeServiceImpl.CORE_MIME_TYPES);
+        loadMimeTypes(MimeTypeServiceImpl.MIME_TYPES);
+
+        // null
+        String mm = null;
+        assertEquals("Extension " + mm, mm, this.service.getExtension(mm));
```

Comment:
I find it hard to understand what this test does. Perhaps split into multiple methods that each validate a "negative" path?

Reply:
I attempted to simplify them